yb-voyager/src/utils/utils.go

```diff
@@ -337,6 +337,37 @@ func ContainsAnySubstringFromSlice(slice []string, s string) bool {
 	return false
 }
 
+func PollForMessageFromOffsetInFile(filePath string, offset int64, message string) error {
```

Comment:
Let this func be specifically for log files. (generically, you will have to think about multiline searches as well and so on). 

You can call this `WaitForLineInLogFile`